scripts/setup-ocs-ci.sh

```diff
@@ -36,9 +36,9 @@ if [ "$OCS_CI_ON_BASTION" == true ]; then
 	exit $ocs_ci_on_bastion_rc
 fi
 
-sudo yum -y install libffi-devel lapack atlas-devel openssl-devel gcc gcc-c++ gcc-gfortran make patch
-sudo yum -y install python38-devel python38-setuptools python3-virtualenv python3-docutils rust-toolset
-sudo yum -y install curl libcurl-devel unzip libxml2-devel
+sudo dnf -y install libffi-devel lapack atlas-devel openssl-devel gcc gcc-c++ gcc-gfortran make patch
+sudo dnf -y install python3-devel python3-setuptools  rust-toolset
```

Comment:
It should be `sudo dnf -y install python39-devel python39-setuptools  rust-toolset` , otherwise setup script is failing with following error: 
```
line 83: python3.9: command not found
``` 
As python3 will install Python 3.6 from the python36 module.

Reply:
In RHEL 9, there is no python39-devel python39-setuptools packages 

`dnf -y install python3-devel python3-setuptools` installs  `python3.9`

```
[root@ocp4alll338 ~]# dnf -y install python39-devel python39-setuptools  rust-toolset
Failed to set locale, defaulting to C.UTF-8
Updating Subscription Management repositories.
Last metadata expiration check: 1:31:11 ago on Mon Feb 13 22:40:13 2023.
No match for argument: python39-devel
No match for argument: python39-setuptools
Package rust-toolset-1.62.1-1.el9.ppc64le is already installed.
Error: Unable to find a match: python39-devel python39-setuptools
[root@ocp4alll338 ~]#

```

Reply:
Got it. Apologies, I tested it on RHEL8 and when I executed `dnf -y install python3-devel` it installed python3.6